app.js

```diff
@@ -1,5 +1,39 @@
 // import functions and grab DOM elements
+import { countAsAYes } from './utils.js';
+
+
+const quizButton = document.getElementById('quiz-button');
+const resultsDiv = document.getElementById('results-div');
+const infoSection = document.getElementById('info');
+const listSection = document.getElementById('lists');
 
 // initialize state
+let correctAnswers = 0;
+
+// set event listeners to update state and DOM
+quizButton.addEventListener('click', () => {
+    alert('Welcome to the Alchemy Campaign!');
+
+    const confirmation = confirm('Are you sure you want to continue this adventure?');
+
+    if (!confirmation) return;
+
+    const firstName = prompt('What is your characters first name?');
+    const lastName = prompt(`What is ${firstName}\'s last name?`);
```

Comment:
Looks like this little "\" is the only reason you're not passing CI. I think "\" only works in a 'string like this' and `not like this` if I'm not mistaken.